js/src/launchAsync.ts

```diff
@@ -149,7 +151,9 @@ export function launchAsync(token: string, subdomain: string, content: Content,
                     disableFirstRun: options.disableFirstRun,
                     readAloudOptions: options.readAloudOptions,
                     translationOptions: options.translationOptions,
-                    displayOptions: options.displayOptions
+                    displayOptions: options.displayOptions,
+                    sendPreferences: !!options.onPreferencesChanged,
```

Comment:
What does !! do? Make sure it is a bool?

Reply:
Yep, it returns true if onPreferencesChanged is not null, false otherwise.